samples/peripheral/radio_test/README.rst

```diff
@@ -0,0 +1,189 @@
+.. _radio_test:
+
+Radio Test
+##########
+
+The Radio Test sample demonstrates how to configure the radio in a specific mode and then test its performance.
+The sample provides a set of predefined commands that allow you to configure the radio in three modes:
+
+* Constant RX or TX carrier
+* Modulated TX carrier
+* RX or TX sweep
+
+Overview
+********
+
+The tests are controlled with Command Line Interface (CLI) through the serial port.
```

Comment:
Is CLI a module in NCS?
If not, isn't what we want to say rather something like:
```suggestion
You can run the tests by connecting to the board through the serial port and sending shell commands.
```

Reply:
Yes, I am using word CLI to refer to shell subsystem from Zephyr.
I am okay with the change if it sounds better to you.

Reply:
Well, there is no module called CLI in Zephyr, so we should use the term that Zephyr uses (namely shell).

Reply:
This is aligned now